src/reset.css

```diff
@@ -0,0 +1,82 @@
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+}
```

Comment:
check the previous comments

Reply:
Hi, it's third time when mentors mentioned to fix this comment
https://github.com/mate-academy/layout_moyo-header/pull/2260/files#r1087531596
if you have any questions, instead of re-requesting review - ask it in the chat